<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentsSettingsWidget</class>
 <widget class="QWidget" name="DocumentsSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="billDefaultTextLabel">
       <property name="text">
        <string>Bill Text:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leBillText"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="offerDefaultTextLabel">
       <property name="text">
        <string>Offer Text:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leOfferText"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateFormatLabel">
       <property name="text">
        <string>Date Format:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leDateFormat"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="footerLabel">
       <property name="text">
        <string>Footer:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="textEditFooter"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnColor">
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Print:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QListWidget" name="lwPrint">
       <item>
        <property name="text">
         <string>Logo</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sender</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beam</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Footer</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Signature</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="envelopeLogoLabel">
       <property name="text">
        <string>Envelope Logo:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnLogo">
         <property name="text">
          <string>Choose...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblEnvelopeLogo">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
